### PR TITLE
feat(docs): add source and contributors

### PR DESCRIPTION
**Pull request by sxzz**
closes #6038

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

Reference:
- https://github.com/vueuse/vueuse

### CODE REVIEW

**Comment by JeremyWuuuuu** [comment]
Before I dive into the code review I found some issues on the preview website: 

image

**Comment by JeremyWuuuuu** [comment]
image

Also there are bugs for the ToC component on the right side.

File: docs/.vitepress/vitepress/components/globals/contributors.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          target="_blank"
+          class="flex gap-2 items-center link"
+        >
+          <img
+            :src="`https://gravatar.com/avatar/${c.hash}?d=retro`"
+            class="w-8 h-8 rounded-full"
+          />
+          {{ c.name }}
+        </a>
+      </div>
```
Comment (JeremyWuuuuu) [inline docs/.vitepress/vitepress/components/globals/contributors.vue:30]: Should we use the component created before in `.vitepress/vitepress/components/common/vp-link.vue`?

**Review by jw-foss — COMMENTED** [review]
Most of the code made sense to me, still I left a few comments for you to take care of, let me know if you have any questions :)

**Comment by sxzz** [comment]
`Last Updated:Invalid Date`
This bug already exists in in staging docs.

**Comment by JeremyWuuuuu** [comment]
@sxzz seems like you have a conflicted file to be resolved, lets get this merged ASAP.

**Comment by sxzz** [comment]
@JeremyWuuuuu The current solution still has unresolved issues. Should we use the local Git or the GitHub API?
If we use the GitHub API, I need a Graphql statement.
If using local Git, how do I get the GitHub `login_name`?

**Review by buqiyuan — APPROVED** [review]

**Pull request merged**